src/main/java/com/google/sps/data/Candidate.java

```diff
@@ -34,39 +35,37 @@ public static Builder builder() {
     return new AutoValue_Candidate.Builder();
   }
 
-  // creates a new Candidate object by extracting the properties from "obj"
-  public static Candidate fromJSONObject(JSONObject obj) throws JSONException {
-    return Candidate.builder()
-        .setName(obj.getString("name"))
-        .setPartyAffiliation(obj.getString("party"))
+  @AutoValue.Builder
+  public abstract static class Builder {
+    public abstract Builder setName(String name);
+
+    public abstract Builder setPartyAffiliation(String partyAffiliation);
+
+    public abstract Builder setCampaignSite(String campaignSite);
+
+    public abstract Builder setPlatformDescription(String platformDescription);
 
+    public abstract Candidate build();
+  }
+
+  // creates a new Candidate object by extracting the properties from "candidateData"
+  public static Candidate fromVoterInfoQuery(JSONObject candidateData) throws JSONException {
+    return Candidate.builder()
+        .setName(candidateData.getString("name"))
+        .setPartyAffiliation(candidateData.getString("party"))
         // TODO(caseyprice): get values for campaignSite and platformDescription
         .setCampaignSite("")
         .setPlatformDescription("")
         .build();
   }
 
   // creates a new Entity and sets the proper properties.
```

Comment:
Update this comment to something like "converts the candidate into a datastore entity and puts the entity into the given datastore instance."